uavcan/equipment/esc/1030.RawCommand.uavcan

```diff
@@ -1,7 +1,9 @@
 #
-# Raw ESC command normalized into [-8192, 8191]; negative values indicate reverse rotation.
-# The ESC should normalize the setpoint into its effective input range.
+# Raw ESC command normalized into [-8192, 8191]. The ESC should normalize the setpoint into its effective input range.
 # Non-zero setpoint value below minimum should be interpreted as min valid setpoint for the given motor.
+# Positive values indicate forward rotation, negative values indicate reverse rotation, and zero indicates no rotation.
+# Forward rotation shall be the direction causing forward thrust. For example, a typical quadcopter always commands
```

Comment:
remove "Forward rotation shall be the direction causing forward thrust". "forward" is a problem, eg. copters have no "forward" thrust